pilot/OWNERS

```diff
@@ -2,6 +2,7 @@ approvers:
   - andraxylia
   - ayj
   - costinm
+  - diemtvu
```

Comment:
can you constrain the ownership to specific folders?

Reply:
The code spread out under networking (including core), model, proxy (mostly for debug utility) so it's a bit of a trouble.

Reply:
Maybe shriram is right - the auth plugin is the main component where security should reside. It may be worth moving all pieces of code related to spiffee and auth-config generation to the plugin.